sql/hive/src/main/scala/org/apache/spark/sql/hive/orc/OrcFileFormat.scala

```diff
@@ -58,10 +58,7 @@ class OrcFileFormat extends FileFormat with DataSourceRegister with Serializable
       sparkSession: SparkSession,
       options: Map[String, String],
       files: Seq[FileStatus]): Option[StructType] = {
-    OrcFileOperator.readSchema(
-      files.map(_.getPath.toString),
-      Some(sparkSession.sessionState.newHadoopConf())
-    )
+    org.apache.spark.sql.execution.datasources.orc.OrcFileFormat.readSchema(sparkSession, files)
```

Comment:
I am not sure of this one too. This looks a complete rewrite of `org.apache.spark.sql.hive.orc.OrcFileOperator.readSchema`.. Is this change required to fix this issue?

Reply:
Yes. It's intentional. OrcFileOperator will be replaced later completely. I made this PR as small as possible for review.